blog/2020/08/juliacon-2020-wrapup.md

```diff
@@ -0,0 +1,38 @@
+
+## Outcomes
+
+JuliaCon 2020 had over 10,000 conference registrations this year with over 1,000 of those people joining us on the official conference discussion forum, discord. On top of the 10,000 registrations, we also garnered 26,000+ unique viewers on the Julia Languages YouTube channel during the conference (up from 10.8K unique viewers last year) and over 20,000 unique visitors to the JuliaCon website (a 140% growth from last year's conference). We are also pleased to report that according to Google Analytics, over 21% of the unique visitors were identified as female which may have potentially been as a result of concerted efforts to engage with and invite people from underrepresented groups in tech to attend JuliaCon. This effort will continue and grow into next year's conference with a focus on inviting diverse folks from specific technical backgrounds where Julia can benefit their work. 
+
+
+This year, we created more than [164 YouTube videos for the conference](https://www.youtube.com/playlist?list=PLP8iPy9hna6Tl2UHTrm4jnIYrLkIcAROR) including talks, workshops, and sponsor addresses. This was an increase from [last year's 116 videos](https://www.youtube.com/playlist?list=PLP8iPy9hna6StY9tIJIUN3F_co9A0zh0H). 
+
+
+In order to provide a strong online experience, it was identified that we would need a web app to deliver a unique JuliaCon viewing experience. Travis DePrato and [other community members](https://github.com/JuliaCon/juliacon-webapp/graphs/contributors) were tasked last minute with delivering the web app and they did so brilliantly. https://live.juliacon.org is the URL that hosts the web app and it has been decided that this service will provide a huge benefit to those who may not be able to attend physical JuliaCons in the future (whenever that may be). Thus we plan to keep the web app alive and build on it for next year's conference (regardless of virtual or physical format).
+
+
+As previously mentioned, this year's JuliaCon featured the first iteration of the [JuliaCon Podcast](https://www.youtube.com/watch?v=sECiJ59hPXg) hosted by Keno Fischer, Huda Nassar, and Nathan Daly featuring guests from all over the Julia Community. The videos were also graciously edited by Elliot Saba. 
+
+Being virtual, we didn't want to miss out on the chance encounters, discussions, and general intermingling of attendees that the _hallway track_ usually provides. A dedicated JuliaCon [gather.town][https://gather.town] space was setup that stayed live throughout the event, with scheduled socials each day and a total of 788 unique users entering the space. The ability to form face-to-face group discussions so easily and fluidly facilitated a lot of inclusive discussion and fun that would've otherwise been missed. Thanks to the team at gather.town for hosting us! A minecraft server was also setup... 
```

Comment:
Fix syntax of link to gather.town
```suggestion
Being virtual, we didn't want to miss out on the chance encounters, discussions, and general intermingling of attendees that the _hallway track_ usually provides. A dedicated JuliaCon [gather.town](https://gather.town) space was setup that stayed live throughout the event, with scheduled socials each day and a total of 788 unique users entering the space. The ability to form face-to-face group discussions so easily and fluidly facilitated a lot of inclusive discussion and fun that would've otherwise been missed. Thanks to the team at gather.town for hosting us! A minecraft server was also setup... 
```